internal/beater/config/config.go

```diff
@@ -159,6 +161,6 @@ func DefaultConfig() *Config {
 		AgentAuth:             defaultAgentAuth(),
 		JavaAttacherConfig:    defaultJavaAttacherConfig(),
 		WaitReadyInterval:     5 * time.Second,
-		MaxConcurrentDecoders: 200,
+		MaxConcurrentDecoders: 128,
```

Comment:
Should we set this to zero by default, with zero meaning it is set based on memory?